docs/manage/enabling_surveys_using_config_files.md

```diff
@@ -218,7 +218,7 @@ or
 
 2. Start the emulator and uninstall/reinstall your `em-devapp.app` so that it opens up with the config login screen and prompts you to scan into a Study/Program.
 
-3. Open Safari in the emulator and paste in your `emission://join_study?label=YOUR_STUDY_OR_PROGRAM_LABEL&source=github`, and it should automatically open up e-mission and your config will successfully be loaded into the app with the study/program you've created.
+3. Open Safari in the emulator and paste in your `emission://join_study?label=YOUR_STUDY_OR_PROGRAM_LABEL&source=github`, and it should automatically open up e-mission and your config will successfully be loaded into the app with the study/program you've created. _If you are using an emulator with Google Chrome, this will **NOT** work. You must click on the link, by starting up the docker-compose of [`nrel-openpath-deploy-configs`](https://github.com/sebastianbarry/nrel-openpath-deploy-configs/tree/surveys-info-and-surveys-data) and navigating to `http://localhost:9090/` to see a list of available links found in the `https://github.com/sebastianbarry/nrel-openpath-deploy-configs/blob/surveys-info-and-surveys-data/survey-resources/index.js`_
```

Comment:
Have you actually tried this? (navigating to `http://localhost:9090`) 
I think you will find that it doesn't actually work

Reply:
For Android emulator, we use `10.0.2.2` to access your local device so I fixed this in the document in https://github.com/e-mission/e-mission-docs/pull/847/commits/ab97fa96429bc303e415d7eac1b88164aa7d1d62